checkmate/async/celery.py

```diff
@@ -47,7 +52,16 @@ def bootstrap_worker(sender, **_kwargs):  # pragma: no cover
     """Set up the celery worker with one-time initialisation."""
 
     # Put some common handy things around for tasks
-    checkmate = create_app(celery_worker=True)
+    try:
+        checkmate = create_app(celery_worker=True)
+
+    # pylint: disable=broad-except
+    except Exception as err:
```

Comment:
You can put it on the same line so that it only applies to that line (not the entire rest of the function):

```suggestion
    except Exception as err:  # pylint: disable=broad-except
```

Reply:
I tried that first, but it didn't want to play for some reason... although now it seems fine!